app/views/govuk_publishing_components/components/docs/big_number.yml

```diff
@@ -0,0 +1,75 @@
+name: Big number
+description: A big number, with a label if needed, for visualising quantitative values such as the number of government departments or historical prime ministers.
+body: |
+  This component needs at least one item in the `items` array passed to it, otherwise it won't render. If no `number` attribute is passed in the item, it will default to "0".
```

Comment:
This isn't strictly true... if I wanted to display a number 0 and nothing else, I have to pass 0 as the number attribute, otherwise the component considers the data empty and doesn't render anything.

Maybe this is trying to be too clever... why not just display whatever is passed, and if nothing is passed, don't display anything?

Reply:
I probably should've been clearer here. The 2nd sentence is for if, for some reason, other attributes like `label` are passed by the `number` attribute isn't. I suppose I could just check for the number and render nothing if it's not there?

Reply:
Sounds good.